babble/src/androidTest/java/io/mosaicnetworks/babble/node/BabbleNodeTest.java

```diff
@@ -73,182 +79,4 @@ public Block onReceiveBlock(Block block) {
         assertArrayEquals(sendBytes, rcvBytes);
     }
 
```

Comment:
Were the extra tests here superfluous, or are we going to add them back in later?

Reply:
They weren't superfluous. We'll add them back in at some point.